data/categories.txt

```diff
@@ -3253,7 +3253,6 @@ nb:Galge
 nl:Galg
 pl:Szubienica
 pt:Forca
-pt-BR:Galo
```

Comment:
To confirm: pt-BR translations should be equal to pt, right? That's why they're removed here. Right?

Reply:
Exactly! Both strings are equal to pt, so that's why I deleted them.